spec/unit/generator/xcconfig/pod_xcconfig_spec.rb

```diff
@@ -133,13 +137,6 @@ module XCConfig
             @xcconfig.to_hash['PODS_TARGET_SRCROOT'].should == '${PODS_ROOT}/../../spec/fixtures/banana-lib'
           end
 
-          it 'adds the library build headers and public headers search paths to the xcconfig, with quotes' do
```

Comment:
Can/should this be converted to a negative test?

Reply:
I like it, I may have toyed with the idea but ultimately decided to remove. Will add back.